randovania/games/prime1/exporter/scan_tips.py

```diff
@@ -0,0 +1,696 @@
+        "&main-color=#FF3333;Hidden message discovered.",
+        "",
+        (
+            "&main-color=#89D6FF;It is possible to climb up to this room's top door from the bottom. "
+            "There are multiple methods, the easiest of which only needs the "
+            "&main-color=#FF3333;Space Jump Boots&main-color=#89D6FF; to climb the crashed frigate's exterior."
+        ),
+    ],
+    "3737672425": [
+        # Monitor Station climb tip
+        # Replaces Monitor Station 2f left computer scan
+        "&main-color=#FF3333;Hidden message discovered.",
+        "",
+        (
+            "&main-color=#89D6FF;By jumping along the rectangular extrusions on this building's exterior, "
+            "it is possible to reach the roof without Space Jump Boots."
+        ),
+    ],
+    "3893588753": [
+        # Monitor Station climb tip
+        # Replaces Monitor Station 2f middle computer scan
+        "&main-color=#FF3333;Hidden message discovered.",
+        "",
+        (
+            "&main-color=#89D6FF;While on this building's roof, it is possible to reach Warrior Shrine without "
+            "lowering the bridge. To do so, stand on top of the roof computer's tallest monitor, "
+            "lock onto a Puffer to the right or left, and perform a lock-on dash towards the ledge."
+        ),
+    ],
+    "1839087853": [
+        # Tree Weakness Tip
+        # Replaces Main Plaza tree scan
+        "&main-color=#FF3333;Hidden message discovered.",
+        "",
+        (
+            "&main-color=#89D6FF;Opening the tree requires a &main-color=#FF3333;Super Missile&main-color=#89D6FF;."
+            "\n\n\nAdditionally, it is possible to peek the item inside the tree without opening it. "
+            "To do so, stand at a specific angle and location on the right side of the tree's lip, then jump in place."
+        ),
+    ],
+    "1530752134": [
+        # Phendrana Canyon L-Jump Tip
+        # Replaces Phendrana Canyon platform scan
+        "&main-color=#FF3333;Hidden message discovered.",
+        "",
+        (
+            "&main-color=#89D6FF;The item in this room can be acquired without the Scan Visor "
+            "by using the &main-color=#FF3333;Space Jump Boots&main-color=#89D6FF; and L-Jumping."
+        ),
+    ],
+    "3740523398": [
+        # Phendrana Canyon Escape Tip
+        # Replaces Phendrana Canyon half-pipe fish scan
+        "&main-color=#FF3333;Hidden message discovered.",
+        "",
+        (
+            "&main-color=#89D6FF;As long as the boxes in this room remain intact, "
+            "they can be climbed to exit this room without any items."
+        ),
+    ],
+    "442524052": [
+        # Rapid Fire Missiles Tip
+        # Replaces Main Plaza holobanner
+        "&main-color=#FF3333;Hidden message discovered.",
+        "",
+        (
+            "&main-color=#89D6FF;Hold&image=5D9F9796;and rapidly tap "
+            "&image=EA2A1C5C; to fire missiles as fast as possible."
```

Comment:
This isn't a trick and is a slightly arbitrary location to have the hint so I would remove it.

Reply:
This fits in goals 2 & 3, and I think Main Plaza is a pretty good place for beginner-focused, low-inventory-requirement tips. I don't want to remove it, but I'm open to alternate suggestions as to where to put it.